### ISSUE TITLE
[BUG] Cathing pokemon nearby on walk

### ISSUE THREAD

**Issue by flo-security** [Panthere/GoBot#110]
I set the 11th release bot to catch pokemon nearby in a radius of 200 m and my speed was 30 kph. 2 or 3 times it does its job great, but after that it starts doing weird things. I can't really explain it, but here is the issue. At some point it does NOT go to the initial route. When I click stop, It seems like it is stuck in the same loop.
![setting](https://cloud.githubusercontent.com/assets/20738058/17269645/947c5c7e-564e-11e6-8ab3-807cf3171d20.PNG)
![bug](https://cloud.githubusercontent.com/assets/20738058/17269646/97fc217c-564e-11e6-9407-d55529391775.PNG)

**Comment by Panthere** [comment]
It 'walks' back to the initial route when it has moved from its previous position before it checked for pokemon, very strange...
I am looking into it and the problem with 'overstepping' pokestops/pokemon on high speeds.

**Comment by flo-security** [comment]
Thanks! In the meantime I will be botting w/o using "Catch Pokemon nearby on Walk"

**Comment by Panthere** [comment]
I've actually not had it happen in every single test I have performed.
I always have 'teleport to pokemon' enabled though.

**Comment by flo-security** [comment]
Well. As you can see, I had it turned off. Maybe I will turn on ''teleport to pokemon''

**Comment by flo-security** [comment]
Then I will turn on teleport to pokemon, As you can see I had it turned off.

**Comment by flo-security** [comment]
![image](https://cloud.githubusercontent.com/assets/20738058/17270169/c4212bf4-565c-11e6-9082-cc78ce3d5414.png)
![image](https://cloud.githubusercontent.com/assets/20738058/17270182/25de9b7e-565d-11e6-8ad7-a64dbdd9e044.png)

Turning on TP to pokemon give me these results all the time. Im now afraid that it softbanned me. Well it isnt really a problem because in 2h-3h i can catch pokemon again. 
I am using the newest .net 4.6.1 with Windows 7 and I have checked Windows Update for any Net framework updates.

**Comment by Panthere** [comment]
Keep the catch radius at 100 and see if it fucks up.
Very concerning...
Shouldn't softban, I also notice you use Google directions, might be it.
Will run with your settings and see if I can pick the point it screws up.

**Comment by flo-security** [comment]
![image](https://cloud.githubusercontent.com/assets/20738058/17270201/caf6f0ac-565d-11e6-88de-b5bb51010d7b.png)
Exact position of where I am farming. Obviously lots of pokemon.

**Comment by flo-security** [comment]
Cant test 100 m setting right now. Got that server related login problem mentioned in the post earlier right now. Will try at 18:00 CET.

**Comment by Panthere** [comment]
I've been running in Adelaide, Australia for the past hour (your settings) 1 invalid packet response (probably timed out authentication). Running on PTC. No stuff ups with the routes yet...

**Comment by flo-security** [comment]
For now, it seems it has something to do with the servers here in Holland. Its out right now here in West Europe. Just logged in in my phone. No pokemon or pokestops visible. I do think that could be the case why it did work at first. I will test it ASAP

**Comment by Panthere** [comment]
Zero issues after 3 hours, must be servers... :(

**Comment by flo-security** [comment]
Now working! Catching pokemon with 30kph, TP to pokemon and farming pokestops like a charm and 200m radius. Only thing to mention is the spam of "Catching Pokemon". Do not know if this is normal.

![image](https://cloud.githubusercontent.com/assets/20738058/17271768/47038a9e-5685-11e6-89d4-15d6de0ae44d.png)
![image](https://cloud.githubusercontent.com/assets/20738058/17271770/561d839a-5685-11e6-97b0-04ff0b1748d3.png)
(Using home city now)

Really, really good bot. Owning every gym in the vicinity ;)
Keep up the good work man! Awesome program

**Comment by NYF2** [comment]
Was it established that the 'catching 0 pokemon' was simply a server issue? I am experiencing same issue with my bot in santa monica. However, I am in east coast.

![capture1a](https://cloud.githubusercontent.com/assets/20744246/17272207/6e3f8652-565e-11e6-829c-98a84a2b75af.PNG)
![capture1b](https://cloud.githubusercontent.com/assets/20744246/17272208/6fe8d616-565e-11e6-8875-935602c53224.PNG)

**Comment by flo-security** [comment]
I had the same issue in NYC Cantral Park, however servers are unstable atm. When I tried again I switched to Utrecht (Netherlands) and everything seemed fine

**Comment by NYF2** [comment]
I know Santa Monica Pier is suppose to be a hot spot and it's mid-day on a saturday, so I'll try elsewhere. I'm guessing you waited awhile before restarting bot? I figure Niantic can recognize a massive leap in user locations.

**Comment by flo-security** [comment]
Yes. I waited a hour before switching continent. Otherwise softban 3h. Very annoying.
Its very absurd since Panthere didnt have any problems at all. 
It could be server problem, Because I had login problems (farming 0 pokestop) this afternoon (1400-1800h Central European Time)  it could be bot bug. Since it spams a lot of "Catching Pokemon", even when there is no one in the vicinity.

**Comment by NYF2** [comment]
Very interesting that you jumped continents in an hour and they didnt catch it. I imagines they would auto recognize leaps by general  time or red flag leaps based on geographical changes under a certain amount of time. What level are you and what are your pokemon settings? I wanted to keep evolves strictly to caterpie, pidgey, and weedle since they use the least candies and I dont care about them. I also transfer anything under 400 CP, because I'm level 23 and i find level 500CP pretty regularly. But a lowest stage pokemon dont ever really come around above 400, so thats why i kept it there.

**Comment by flo-security** [comment]
My settings are no evolves, catch all, under CP 300 and/or IV <60% gets transferred right away. I am level 20.

**Comment by NYF2** [comment]
going to try santa monica one more time while i'm at work. maybe itll settle towards sun down...

**Comment by flo-security** [comment]
Just tested Santa Monica Pier. It does not work for me either. It catches around 4 or 5 pokemon. Then it loops and doesnt go to a pokestop to farm. I guess it is something serious...

**Comment by flo-security** [comment]
I will leave it to tomorrow. Hopefully the servers will be refined. Since I heard on the pokemon go twitter accounts there was lots of pokestop updates etc. I hadnt had any on the map for 2/3 hours today, same story for  pokemon.
I am confident that it will be resolved soon.

**Comment by NYF2** [comment]
Just got off work and in 9 hours I did about 50-60K XP, found 300 pokemon, transferred 185 (under 400 CP) and as I type this it's saying 7,500 XP/H with 28.3 pokemon/hour. My star dust is up to 80,000 though! room to make some powerhouse pokemon now. Overall, not too bad...could be better though. I would like to have 15,000 XP an hour, but I'm only evolving 5 pokemon. Do you know what specific locations are suppose to be the best?

**Comment by Phoo7** [comment]
Been working great for me really.
My only issue with catch nearby pokemon is that it seems to go off router to much and hitting pokestops is quite low, so I'm running out of balls!

Can't complain though

**Comment by flo-security** [comment]
Now working like a charm, maybe it was servers yesterday. Now farming in Santa Monica Beach like a charm. Only one issue; Not farming pokestops very often. I will make a sort of general discussion here so we can exchange good places to 'find' pokemon

**Comment by NYF2** [comment]
I had the same experience with poke stops in santa monica. I'm experiencing the same thing in central park now too with limited pokestop farming. Perhaps a weight attribute can be added to provide a ratio of priority? 

There are reddit discussions on localized areas for pokemon, but I wonder if there is a national map with big nests for each pokemon? I'm sure there is one location where finding each specific pokemon is easier regardless of type. I will look into this.
